types/enzyme-adapter-react-15/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for enzyme-adapter-react-15 1.0
+// Project: https://github.com/airbnb/enzyme
+// Definitions by: Tanguy Krotoff <https://github.com/tkrotoff>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare class ReactFifteenAdapter {
+}
```

Comment:
Oh, and tiny bit of advice. If you'd like the class to be nominal-ish (as it has no members, so as it stands would match anything), you can add a private like `private createRenderer;` to make the class get compared nominaly-ish. Though, I imagine the ideal would actually be to say that the class `extends EnzymeAdapter` after importing `EnzymeAdapter` from `"enzyme"`, then tightening up the type definition of `configure`. Not needed to get the PR merged, though; just a suggestion for improvement!